### PR TITLE
Added fix for sort by name case sensitive issue

### PR DESCRIPTION
**Pull request by paraschhabra96**

### CODE REVIEW

File: graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasJanusGraphManagement.java [hunk 1]
```diff
@@ -193,15 +194,24 @@ public AtlasEdgeLabel getEdgeLabel(String label) {
     }
 
     @Override
-    public String addMixedIndex(String indexName, AtlasPropertyKey propertyKey, boolean isStringField) {
+    public String addMixedIndex(String indexName, AtlasPropertyKey propertyKey, boolean isStringField, String normalizer) {
```
Comment (dggs123) [inline graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasJanusGraphManagement.java:197]: Here you could've added a new method with different parameters, and keep this as it is. But for now its okay.

Comment (paraschhabra96) [reply graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasJanusGraphManagement.java:197]: This is what I have done. Check below.

File: repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java [hunk 2]
```diff
@@ -742,12 +754,17 @@ public String createVertexIndex(AtlasGraphManagement management, String property
                         LOG.debug("Creating backing index for vertex property {} of type {} ", propertyName, propertyClass.getName());
                     }
 
-                    indexFieldName = management.addMixedIndex(VERTEX_INDEX, propertyKey, isStringField);
+                    if (normalizer == "") {
```
Comment (dggs123) [inline repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java:757]: normalizer can also be null

Comment (dggs123) [reply repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java:757]: Also i think you've already handled the null condition in addMixedIndex, so no need to do it here.

File: webapp/src/main/java/org/apache/atlas/Atlas.java [hunk 3]
```diff
@@ -127,12 +143,56 @@ public static void main(String[] args) throws Exception {
...
+            GetIndexTemplatesResponse getTemplateResponse = esClient.indices().getIndexTemplate(request, RequestOptions.DEFAULT);
+            LOG.info("Got get template response");
+            if (getTemplateResponse.getIndexTemplates().size() != 0) {
+                LOG.info("Atlan Index Template already present!");
+            }
+        } catch (ElasticsearchStatusException e) {
+            if (e.status().getStatus() == 404) {
+                LOG.info("Index template not present. Creating...");
+                PutIndexTemplateRequest indexTemplateRequest = new PutIndexTemplateRequest("atlan_template");
+                indexTemplateRequest.source("{\r\n  \"index_patterns\": [\"*janusgraph*\"],\r\n  \"settings\": {\r\n    \"analysis\": {\r\n      \"normalizer\": {\r\n        \"lowerasciinormalizer\": {\r\n          \"type\": \"custom\",\r\n          \"filter\":  [ \"lowercase\", \"asciifolding\" ]\r\n        }\r\n      }\r\n    }\r\n  }\r\n}", XContentType.JSON);
```
Comment (dggs123) [inline webapp/src/main/java/org/apache/atlas/Atlas.java:175]: Its better if we can get this from file.

**Review by dggs123 — APPROVED** [review]

**Pull request merged**